Library/Formula/imagejs.rb

```diff
@@ -0,0 +1,19 @@
+class Imagejs < Formula
+  homepage "http://jklmnn.de/imagejs/"
+  url "https://github.com/jklmnn/imagejs/archive/0.7.1.tar.gz"
+  sha1 "4c3e1c2134194cced5924c9cc577d36165548575"
+  head "https://github.com/jklmnn/imagejs.git"
+
+  def install
+    system "make"
+    bin.install "imagejs"
+  end
+
+  test do
+    touch "test.js"
+    system "echo", "\"alert('Hello,", "World!')\"", ">", "test.js"
+    system "#{bin}/imagejs", "bmp", "test.js", "-l"
+    rm "test.js"
```

Comment:
No need to clean up after the test - Brew will wipe out any trace of the test automatically once it's finished :).